front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -218,12 +252,19 @@ const StdcmConsist = ({ isDebugMode, disabled = false }: StdcmConsistProps) => {
           type="number"
           min={0}
           value={totalMass ?? ''}
-          onChange={onTotalMassChange}
+          onChange={(e) => {
+            onTotalMassChange(e);
+            clearMissingErrorIfNotEmpty('totalMass', e.target.value);
+          }}
           onBlur={() => {
-            setConsistErrors({
-              ...consistErrors,
-              totalMass: { ...consistErrors.totalMass, display: !!consistErrors.totalMass.message },
-            });
+            setConsistErrors((prev) => ({
+              ...prev,
+              totalMass: {
+                message: totalMassError,
+                display: !!totalMassError,
+                type: totalMassError === t('consist.errors.missingValue') ? 'missing' : 'invalid',
+              },
+            }));
```

Comment:
The 3 `onBlur` are pretty similar, we could create a function for that : 
```
const handleBlurError = (field: keyof ConsistErrors, error?: string) => {
  setConsistErrors((prev) => ({
    ...prev,
    [field]: {
      message: error,
      display: !!error,
      type: error === t('consist.errors.missingValue') ? 'missing' : 'invalid',
    },
  }));
};

```
```suggestion
          onBlur={() => handleBlurError('totalMass', totalMassError)}
```

Reply:
Thanks, it's clearer now !